modules/ROOT/pages/access-nosql-databases.adoc

```diff
@@ -13,18 +13,18 @@
 :page-type: general
 = Access NoSQL databases with CDI
 
-Not only SQL (NoSQL) databases provide flexible schemas for potentially large amounts of structured, semi-structured, and unstructured data.
-You can access a NoSQL database with a Contexts and Dependency Injection (CDI) producer in Open Liberty, without enabling features like MongoDB or CouchDB.
+In Open Liberty, you can create a CDI producer and configure access to any version of a NoSQL Database in your application.
+With a CDI producer, you are not restricted to certain versions of NoSQL databases to provide a database to your application.
 
-Before Open Liberty version 19.0.0.6, using MongoDB or Couch DB databases required you to enable the MongoDB or CouchDB feature.
-The Mongo DB feature was stabilized in version 19.0.0.4 and the CouchDB feature was stabilized in version 19.0.0.6.
-To connect to these databases in version 19.0.0.6 and later, you can https://openliberty.io/guides/cdi-intro.html[configure a CDI producer in your application code].
+A CDI producer can be modified according to your needs and you can configure the database driver by using MicroProfile Config.
+Using a CDI producer eliminates the need to enable features like the MongoDB feature for https://www.mongodb.com/[MongoDB] or the CouchDB feature for https://couchdb.apache.org/[CouchDB].
```

Comment:
for both mongo db and couch db features, use the macro to link and the full text name of the features,  the [MongoDB Integration feature](https://www.openliberty.io/docs/21.0.0.8/reference/feature/mongodb-2.0.html) and the [CouchDB Integration feature](https://www.openliberty.io/docs/21.0.0.8/reference/feature/couchdb-1.0.html).